Checkpoint/checkpoint/ingest/parser.yml

```diff
@@ -168,6 +168,7 @@ stages:
           url.full: "{{checkpoint.message.request}}"
           url.original: "{{checkpoint.message.request}}"
           user_agent.original: "{{checkpoint.message.user_agent}}"
+          action.tcp.flag: "{{checkpoint.message.get('TCP Flags')}}"
```

Comment:
To be consistent with others formats (like [Netflow](https://github.com/SEKOIA-IO/intake-formats/blob/main/NetFlow/netflow/ingest/parser.yml#L237) or [OpenBSD packet filter](https://github.com/SEKOIA-IO/intake-formats/blob/main/OpenBSD/openbsd-packet-filter/ingest/parser.yml#L48)) , I wonder if we shouldn't declare a custom field relative to checkpoint (like `checkpoint.firewall.tcp_flags`)?